src/containers/App.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+
+import Main from './Main';
+
+const App: React.FC = () => {
+  return (
+    <BrowserRouter>
+      <Switch>
+        <Route exact path="/" component={Main} />
```

Comment:
라우트가 하나뿐인데 react-router-dom을 쓰신 이유가 있을까요??

Reply:
처음에 Main 페이지로 store를 inject했을 때, Provider를 달고도 따로 store를 주입해야하더라구요.
근데 react-router-dom을 쓰면 별 다른 처리를 하지 않아도 되서 그렇게 처리했는데
이렇게 수정할 계획입니다
```suggestion
   const injected = props as InjectedProps;
   
   const { store } = injected;
```